frontend/src/features/upload/components/LabeledSelectInput.tsx

```
@@ -0,0 +1,69 @@
+          type='text'
+          value={value}
+          onChange={(e) => handleChange(e.target.value)}
+          className={`w-full bg-gray-50 px-5 py-[0.875rem] rounded-lg focus:outline-none relative ${isOpen ? 'z-20' : 'z-0'}`}
+          onClick={() => handleDropdown?.(!isOpen)}
+        />
+        {hasDropdown && isOpen && (
+          <>
+            <div
+              className='fixed inset-0 z-10 cursor-default'
+              onClick={() => handleDropdown(false)}
+            />
```

Biome (2.3.13)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)